src/components/markdown-help.jsx

```diff
@@ -0,0 +1,269 @@
+import React from 'react';
+import Markdown from './markdown';
+import simpleAvatar from '../images/simple-avatar.png';
+
+const TalkMarkdownHelp = () =>
+  (<table>
+    <thead>
+      <tr>
+        <th colSpan="2">Zooniverse Hashtags, Users and Subjects</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Add a Hashtag</td>
+        <td>#hashtag</td>
+      </tr>
+      <tr>
+        <td>Mention User</td>
+        <td>@user</td>
+      </tr>
+      <tr>
+        <td>Mention Subject<br />(project Talk)</td>
+        <td>^S<subject_id /><br />e.g. ^S830273</td>
+      </tr>
+      <tr>
+        <td>Mention Subject<br />(any Talk)</td>
+        <td>@owner/project^S<subject_id /><br />e.g. @zooniverse/wildcam-gorongosa^S830273</td>
+      </tr>
+      <tr>
+        <td>Mention<br />Project Teams</td>
+        <td>
+          @admins - mention the project administrators<br />
+          @moderators - mention the project moderators<br />
+          @researchers or @scientists - mention the project researchers<br />
+          @team - mention the entire Zooniverse team
+        </td>
+      </tr>
+      <tr>
+        <th colSpan="2">Adding Emoji to Posts</th>
+      </tr>
+      <tr>
+        <td>Complete<br />Emoji List</td>
+        <td><a href="http://www.emoji-cheat-sheet.com/" rel="noopener noreferrer" target="_blank">use this website for all supported emoji</a></td>
+      </tr>
+    </tbody>
+  </table>);
+
+const MarkdownHelp = ({ title, talk }) => {
+  const bulletedList = `
```

Comment:
The Markdown component was not rendering multi-line content correctly, so I created the separate variables with string interpolation as a workaround (I tried both with and without string interpolation in the Markdown component, no difference in the result). Below are some screen shots showing the difference. Happy to change this if there's a way to render multiline content in the Markdown component, but so far I haven't found anything on Stack Overflow/google.

![image](https://user-images.githubusercontent.com/12473119/28890683-84f811a6-778d-11e7-8826-1fe45b60a543.png)

![image](https://user-images.githubusercontent.com/12473119/28890700-8f3543a0-778d-11e7-94b5-911d81da629a.png)